l10n_es_aeat_mod190/models/l10n_es_aeat_mod190_report.py

```diff
@@ -320,8 +320,7 @@ def _compute_partner_record_ok(self):
         required=True,
     )
     aeat_perception_subkey_id = fields.Many2one(
-        comodel_name="l10n.es.aeat.report.perception.subkey",
-        string="Perception subkey",
+        comodel_name="l10n.es.aeat.report.perception.subkey", string="Perception subkey"
```

Comment:
This diff is useless

Reply:
Hello @pedrobaeza 
There is a difference between black and flake8 applied in the code

Reply:
Not really if you are using the same versions and don't modify it. Here, you have removed the trailing comma, and that's why it has joined it in the same line.

Reply:
Done